presto-docs/src/main/sphinx/gcp/storage.rst

```diff
@@ -0,0 +1,216 @@
+Querying Google Cloud Storage
```

Comment:
These documentation is specific to the Hive connector. Let's add another section *Google Cloud Platform Configuration* to the Hive connector docs, after the one for Amazon S3.